src/cpptools.ts

```diff
@@ -28,7 +28,10 @@ export class CppConfigurationProvider implements cpp.CustomConfigurationProvider
     private workspaceBrowseConfiguration: cpp.WorkspaceBrowseConfiguration = { browsePath: [] };
 
     private getConfiguration(uri: vscode.Uri): SourceFileConfigurationItem | undefined {
-        const norm_path: string = path.normalize(uri.fsPath);
+      let norm_path: string = path.normalize(uri.fsPath);
```

Comment:
Nit: The spacing/tabbing looks off by a couple of spaces, it looks like it's not indented enough.

Reply:
thx for spotting that. fixed with latest commit.